ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -66,7 +66,7 @@ tests:
     from: src
   skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
 - as: aws-e2e-ccm-install
-  skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  cron: 0 9 * * 1
```

Comment:
I’m curious what time zone the job uses? Is this 9am UTC?

Reply:
Yes. Failures should show up on Monday morning. LMK if I miscalculated the conversion.